### PR TITLE
Add Flutter snap path to getKnownFlutterSdkPaths()

### PR DESCRIPTION
**Pull request by MarcusTomlinson**
Fixes: https://github.com/canonical/flutter-snap/issues/19.

Changes proposed in this pull request:

* If present, add the Flutter snap SDK path to the list returned from getKnownFlutterSdkPaths()
  * This path is well-known and standard on all Linux platforms.

### CODE REVIEW

**Review by devoncarew — CHANGES_REQUESTED** [review]
Thanks for the contribution! One comment on the PR, in-line.

File: src/io/flutter/sdk/FlutterSdkUtil.java [hunk 1]
```diff
@@ -139,6 +139,12 @@ public static void addKnownSDKPathsToCombo(@NotNull JComboBox combo) {
       paths.add(fromUserPath);
     }
 
+    // add the snap SDK path if it exists (standard on all Linux platforms)
+    final File snapSdkPath = new File(System.getenv("HOME") + "/snap/flutter/common/flutter");
```
[the remark this answers is not in the document]
Comment (MarcusTomlinson) [reply src/io/flutter/sdk/FlutterSdkUtil.java]: Ah yes, much cleaner. Done :)

**Review by devoncarew — APPROVED** [review]
Thanks!

**Comment by stevemessick** [comment]
@MarcusTomlinson Could you submit a PR that updates the AUTHORS file with your name and email? I'm working on the release notes for the version we're publishing this Friday and would like to give your credit for your contribution.

**Comment by MarcusTomlinson** [comment]
> @MarcusTomlinson Could you submit a PR that updates the AUTHORS file with your name and email? I'm working on the release notes for the version we're publishing this Friday and would like to give your credit for your contribution.

Absolutely :) Will do so in a few.

**Comment by stevemessick** [comment]
Also, if you have a screen shot to illustrate the feature I'd be happy to include it. I don't have Linux handy.

**Pull request merged**